models/src/main/protobuf/CasperMessage.proto

```diff
@@ -110,6 +121,12 @@ message BlocksResponse {
   int64 length = 3;
 }
 
+message BlockInfosAtHeight {
```

Comment:
I don't understand the point of grouping blocks at the same height when returning the result from `showBlocks`. I know it is grouped like that in the topological sort, but you could just do `dag.topoSort.takeRight(depth).flatten` and then return the `BlockInfoWithoutTuplespace` sequence directly.

Reply:
I thought it would be clearer if it is grouped and thus ordered by height. Otherwise it will be ordered sort of by height but actually randomly.